src/site/content/en/blog/vitals-business-impact/index.md

```diff
@@ -129,16 +129,37 @@ to monitor progress and impact across teams.
 - Experimented with different rendering techniques
 (for example, SSR LCP element vs SSR above the fold content vs Full client-side rendering).
 
+### Redbus
+
+CWV fixes contributed to 80-100% mobile conversion rates(mCVR) and significant domain ranking uplift across Redbus’s global market properties.
+
+{% Img 
+src="image/akX10MMTHtVhXYWSgqNknjak5KQ2/1988aZMNJ7viKhj8U3aS.png", 
+alt="192% domain ranking uplift in Columbia", 
+width="800", height="360" %}
+
+#### Techniques
+
+- Fixing slots for page components and removal of unoptimized tag insertion scripts helped [optimize CLS](/optimize-cls).
+- [3P Script Optimization](/controlling-third-party-scripts) and building microservices with single responsibility principle reduced TTI & TBT significantly.
+
+#### Key learnings
+
+- Reducing CLS from 1.65 to 0 significantly uplifted their domain rankings globally.
+- Reducing TTI + TBT by ~100% boosted the mCVR by 80-100% across global properties.
```

Comment:
Has mCVR increased _by 80-100%_ or increased _to 80-100%_? If mCVR can range between 0-100%, an increase _by 100%_ would mean they started from 0 which seems unlikely. 

Similarly for TTI and TBT: a 100% reduction would mean they got those metrics down to 0. Is that true?

Reply:
1. mCVR increased **BY** 80-100%. just as an example, if they were at 20% earlier, now they are at 40%.
2. TTI and TBT halved. The TTI decreased from ~10s to ~5s. I think we could rephrase this statement to say that TTI and TBT **halved**, boosting the mCVR by 80-100% across global properties. WDYT?

Reply:
Yup we confirmed the actual numbers for mCVR, TTI, TBT. I also understand your perspective to give straightforward numbers and we can do this for TTI and TBT! However, partners typically do not share actual numbers for mCVR externally as it is usually sensitive information. Similarly, Redbus wants to share only relative increase (80-100%) for mCVR.

For TTI and TBT, we've confirmed the latest numbers. TTI improved from ~8s to ~4s and TBT improved from ~1200ms to ~700ms.

I think we can phrase this as:
- Reducing CLS from 1.65 to 0 significantly uplifted their domain rankings globally
- Reducing TTI from ~8s to ~4s and TBT from ~1200ms to ~700ms contributed to an 80-100% increase in mCVR across global properties

WDYT?